lib/src/main/scala/eu/unicredit/swagger/generators/DefaultServerGenerator.scala

```diff
@@ -142,7 +142,11 @@ class DefaultServerGenerator extends ServerGenerator with SharedServerClientCode
         } else if (queryParams.tail.isEmpty) {
           p"$pathParams ? ${queryParams.head}"
         } else {
-          p"$pathParams ? ${queryParams.head} & (..${queryParams.tail})"
+          def extractInfixGen(basePat: Pat, params: List[Pat]): Pat = {
+            if(params.isEmpty) basePat
+            else extractInfixGen(Pat.ExtractInfix(basePat, Term.Name("&"), params.head :: Nil), params.tail)
+          }
+          extractInfixGen(p"$pathParams ? ${queryParams.head}", queryParams.tail)
         }
```

Comment:
This can even simplified more, like this:
```
        val params = queryParams match {
            case Nil => pathParams
            case head :: tail => tail.foldLeft(p"$pathParams ? $head")((a, b) => p"$a & $b")
          }
```

Reply:
Thanks for check this code. I think I should write the code according to your advice.

If I had completed fixing code, I request merge you again.

Reply:
Sorry, already had fixed the code. Thank you.

Reply:
Yes, since they were just a few lines of code, I added them myself.

Thanks again for your valuable contribution!